src/main/java/com/icthh/xm/ms/entity/domain/listener/ProfileElasticSearchListener.java

```diff
@@ -33,7 +40,7 @@ public void init() {
     @PostUpdate
     void onPostPersistOrUpdate(Profile profile) {
         log.debug("Save xm entity to elastic {}", profile.getXmentity());
-        runAfterTransaction(profile.getXmentity(), xmEntitySearchRepository::save);
+        runAfterTransaction(profile.getXmentity(), xmEntityElasticRepository::save);
```

Comment:
when entity will be deleted?

Reply:
sorry, not clear what did u mean here. Entity will be deleted when somebody actually deletes it. I did not remove `onPostRemove()` method